tools/test/Sparkline/SparklineBase.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
```

Comment:
remove this file

Reply:
But this is the one which is required for testing

Reply:
done